src/Components/HomePage/HomePage.jsx

```diff
@@ -1,17 +1,18 @@
 import Header from "./../Header/Header.jsx";
 import Footer from "./../Footer/Footer.jsx";
-import Banners from "./../Main/Banners.jsx";
+import Banners from "./../Banner/Banners.jsx";
 import Categories from "./../Categories/Categories.jsx";
 import bannersData from "./../../data/mocks/es-mx/featured-banners.json";
 import FeaturedProducts from "./../FeaturedProducts/FeaturedProducts.jsx";
+import featuredProdData from "../../data/mocks/es-mx/featured-products.json";
 function App() {
```

Comment:
this component has a duplicated name, I think it should be `HomePage`

```suggestion
function HomePage() {
```